basicsynbio/functions/pdf_instructions.py

```diff
@@ -0,0 +1,409 @@
+from basicsynbio.cam import BasicBuild
+import zipfile
+import os
+import pandas as pd
+from pathlib import Path
+from datetime import datetime
+from reportlab.platypus import (
+    SimpleDocTemplate,
+    ListFlowable,
+    Paragraph,
+    Spacer,
+    Image,
+    PageBreak,
+    KeepTogether,
+    Table,
+)
+from reportlab.lib.pagesizes import A4
+from reportlab.lib.units import cm
+from basicsynbio.utils import (
+    PROCESSED_MATERIALS,
+    PROCESSED_INCUBATION_TIMES,
+    PROCESSED_TEMPERATURE_PROFILE_1,
+    PROCESSED_ASSEMBLY_REACTION_3,
+    PROCESSED_PCR_MACHINE_3,
+    PageNumCanvas,
+    style,
+    style_joint_cell,
+    style_no_split,
+    styleN,
+    styles,
+    clips_data_from_pandas,
+    assembly_data_from_pandas,
+)
+
+
+def pdf_instructions(basic_build: BasicBuild):
+    """Writes information about each clip_data and assembly to
+    two dependent CSV files in the same folder the command
+    is executed.
+
+    Args:
+        path (optional): path to zipped folder of csv files. If none defaults to
+            working directory with a time stamped name, output csvs is created.
+
+    Returns:
+        str: filepath of created pdf
+    """
+
+    import pandas as pd
+    import math
+
+    def calculate_clip_num(assemblies: list, assemblies_per_clip: int = 28):
```

Comment:
Can we move/duplicate the `assemblies_per_clip` argument to the `pdf_instructions` function? Users can then specify how many assemblies each clip should be used in.